packages/web-components/src/components/va-minimal-header/va-crisis-line-modal.scss

```diff
@@ -0,0 +1,171 @@
+$red-60v: #b51d09;
+$red-70v: #8b1303;
+
+$small-desktop-screen: 1008px;
+
+
+.va-crisis-line-container {
+  position: relative;
+  margin: auto;
+  background-color: $red-60v;
+
+  @media(min-width: $small-desktop-screen) {
+    max-width: 97.5rem;
+    background-color: transparent;
+  }
+}
+
+a {
+  color: var(--color-link-default);
+}
+
+button.va-crisis-line {
```

Comment:
I think we need `cursor: pointer` somewhere in here. Currently the cursor remains a cursor when hovering over the button

Reply:
Added